modules/lib/src/main/resources/assets/js/app/ui2/dialog/ConfirmationDialog.tsx

```diff
@@ -81,7 +179,7 @@ const ConfirmationDialogFooter = ({
     const {submitEnabled} = useConfirmationDialog();
 
     const cancel = useI18n('action.cancel');
-    const submit = useI18n('action.submit');
+    const submit = useI18n('action.confirm');
```

Comment:
Yeah, we have submit. Rename all related variables from `submit` to `confirm` too, e.g. `submit`, `onSubmit`, etc.